translations/en-US.all.yaml

```diff
@@ -0,0 +1,17 @@
+- id: "Update the annotations on a resource"
+  translation: "Update the annotations on a resource"
+
+- id: "watch is only supported on individual resources and resource collections - {{.Count}} resources were found"
+  translation:
+    one: "watch is only supported on individual resources and resource collections - {{.Count}} resources was found"
```

Comment:
This is part of the `github.com/nicksnyder/go-i18n` package.  Added links to examples.

(and fixed the string)

I'm open to other suggestions for libraries for doing translation/pluralization.  `go-i18n` was the best I could find.